src/applications/personalization/profile360/vet360/components/PhoneField/EditModal.jsx

```diff
@@ -7,19 +7,23 @@ import AlertBox from '@department-of-veterans-affairs/formation/AlertBox';
 import Vet360EditModal from '../base/EditModal';
 
 class PhoneTextInput extends ErrorableTextInput {
-  componentDidMount() {
-    const wrapper = document.createElement('div');
-    wrapper.className = 'usa-only-phone-wrapper';
+  // componentDidMount() {
+  //   const wrapper = document.createElement('div');
+  //   wrapper.className = 'usa-only-phone-wrapper';
 
-    const inputField = document.querySelector('input.usa-only-phone');
+  //   const inputField = document.querySelector('input.usa-only-phone');
 
-    inputField.parentNode.insertBefore(wrapper, inputField);
-    wrapper.appendChild(inputField);
+  //   inputField.parentNode.insertBefore(wrapper, inputField);
+  //   wrapper.appendChild(inputField);
 
-    const prefixEl = document.createElement('span');
-    prefixEl.innerText = '+1';
-    prefixEl.className = 'usa-only-phone-prefix';
-    inputField.insertAdjacentElement('beforebegin', prefixEl);
+  //   const prefixEl = document.createElement('span');
+  //   prefixEl.innerText = '+1';
+  //   prefixEl.className = 'usa-only-phone-prefix';
+  //   inputField.insertAdjacentElement('beforebegin', prefixEl);
+  // }
+
+  render() {
+    return <ErrorableTextInput {...this.props} />;
```

Comment:
I just commented out the DOM operations. I'll create a follow up ticket to add back the `+1`.